src/components/LandmarkDrawer.tsx

```diff
@@ -62,10 +64,31 @@ function LandmarkDefinition ({ definition }: { definition: string })  {
   return <Text>{definition}</Text>;
 }
 
-function LandmarkAssessment ({ landmark }: {
+function LandmarkAssessment ({ landmark, user }: {
   landmark: Landmark;
+  user: User;
 }) {
   const [score, setScore] = useState<LandmarkScore | undefined>();
+  const [markdown, setMarkdown] = useState<string>("");
+  const { refetch } = trpcNext.map.listLandmarkAssessment.useQuery({
+    userId: user.id,
+    landmark: landmark.名称,
+  });
+  const createLandmarkAssessment = async() => {
+    try {
+      if (score !== undefined) {
+        await trpc.map.createLandmarkAssessment.mutate({
+          landmark: landmark.名称,
+          score: score + 1,
+          markdown: markdown,
+        });
+        void refetch();
+      }
+    } finally {
+      setScore(undefined);
+      setMarkdown("");
```

Comment:
why do you need to reset them?

Reply:
After submitting, the result values should be removed?
Here is a screenshot without reset them:
![Screenshot 2024-10-08 at 10 18 39](https://github.com/user-attachments/assets/9ce01ab6-92ce-490d-8b8c-86947178b479)